share/dictionary/dhcpv4/dictionary.rfc4174

```diff
@@ -0,0 +1,15 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+##############################################################################
+#
+#  RFC4174 -  The IPv4 Dynamic Host Configuration Protocol (DHCP) Option
+#             for the Internet Storage Name Service
+#
+#  $Id$
+#
+##############################################################################
+
+# Internet Storage Name Service
+ATTRIBUTE	iSNS					83	octets
```

Comment:
this is a complex structure, finishing off with `ipaddr array` :(

Reply:
Corrected; as we discussed, there's just an ipaddr array at the end rather than the a1-a4 and b1-b4.